src/commit.c

```diff
@@ -383,24 +384,32 @@ int git_commit_amend(
 	return error;
 }
 
-int git_commit__parse_raw(void *_commit, const char *data, size_t size)
+static int git_commit__parse_internal(git_commit *commit, const char *data, size_t size, git_commit__parse_flags flags)
```

Comment:
Well, I now realize that this really accepts an enum, only. I first thought that this was a bitfield due to its naming (flags instead of flag). So another route to make this a tad easier to grasp might be to just rename it to `git_commit__parse_mode mode`, as it's not really about flags at all